### PR TITLE
Pkgpag

### PR DESCRIPTION
**Pull request by pagnani**
* Added Manifest and Project toml  package config files
* Updated the Compat annotation to new standard.

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+Distributed = "8ba89e20-285c-5b6f-9357-94700520ee1b"
+FastaIO = "a0c94c4b-ebed-5953-b5fc-82fe598ac79f"
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+
+[compat]
+Compat = "≥ 0.70.0"
+FastaIO = "≥ 0.1.2"
+julia = "≥ 0.6.0"
```
Comment (carlobaldassi) [inline Project.toml]:
```suggestion
julia = "1"
```

Comment (pagnani) [reply Project.toml]: Done!
Did not change FastaIO and Compat bounds though.

**Comment by pagnani** [comment]
Travis and AppVeyor work!
Good to go!

File: README.md [hunk 2]
```diff
@@ -50,17 +50,22 @@ Installation
...
 
 julia> Pkg.clone("https://github.com/carlobaldassi/GaussDCA.jl")
 ```
 
-All dependencies will be downloaded and installed automatically.
+Alternatively, using julia version 1.0 or later you can use the new
+package manager. From julia REPL, type <kbd>]</kbd> and:
+
+```
+(@v1.?) pkg> add https://github.com/pagnani/GaussDCA.jl.git
```
Comment (carlobaldassi) [inline README.md]:
```suggestion
(@v1.?) pkg> add https://github.com/carlobaldassi/GaussDCA.jl.git
```

Comment (pagnani) [reply README.md]: oops...

Changed

**Comment by carlobaldassi** [comment]
:+1:

**Pull request merged**